packages/@aws-cdk/aws-appmesh/package.json

```diff
@@ -97,7 +98,8 @@
     "@aws-cdk/aws-iam": "0.0.0",
     "@aws-cdk/aws-servicediscovery": "0.0.0",
     "@aws-cdk/core": "0.0.0",
-    "constructs": "^3.2.0"
+    "constructs": "^3.2.0",
+    "@aws-cdk/aws-acmpca": "0.0.0"
```

Comment:
Please add this so the alphabetical order of the dependencies is preserved, like you did for `"dependencies"`.